wikimedia/wikipedia-ios
# Allow account vanishing requests from App

**Phabricator:** https://phabricator.wikimedia.org/T300080

### Notes
* As [Apple requires that apps allow account deletion from all apps in the app store](https://developer.apple.com/news/?id=12m75xbj), we are adding functionality to allow users to initiate the vanishing process from the app (vanishing is the closest we have to account deletion in Wikipedia).
* It prompts a form built in SwiftUI that captures the logged-in user's username and allows them to add any information they would like. It then opens the email client, as the process is done by administrators on Wikipedia and is not automated.

### Test Steps
1. Go to Settings -> Account -> Vanish account.
2. Add information to the `Additional information` field
3. Click send request (it will make you leave the app and open the email client)
5. Please do not send the email (unless you want your account deleted)
6. As you return to the Wikipedia app, it should show a modal with more information

### Screenshots/Videos
